slack_bot/token_rotation.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module contains implementation of app config token rotation logic
+"""
+
+import logging
+import httpx
+from dotenv import find_dotenv, load_dotenv, set_key, dotenv_values
+from pydantic import SecretStr
+from pydantic_core import ValidationError
+
+dotenv_path = find_dotenv()
+load_dotenv(dotenv_path)
```

Comment:
I believe this is not needed, pydatntic load env variables automatically

Reply:
During tests sometimes environment variables could not be found, but maybe now it is redundant
Still, I'd keep this code